containers/runner/launch

```diff
@@ -60,12 +61,13 @@ Options:
  --recommended-jobs                   Print the number of jobs (--jobs option) used
                                       by default and exit. It is calulated from the number
                                       of CPUs and the size of RAM.
+ --scenario NAME                      Name of the running scenario, used for json logs
```

Comment:
I think launcher should not be aware / use "scenario" concept which comes from the GitHub workflow and scenario script. I'd just use something generic like --name (or --id or maybe something better) here (and in the related variables).

Reply:
Sure, it's there so that the name can be added to the json summary to group them together.

Reply:
I changed it to --group, so that the results can be grouped together by scenario name when imported into an analysis tool.